src/max_key.ts

```diff
@@ -8,7 +10,7 @@ export interface MaxKeyExtended {
  * @public
  * @category BSONType
  */
-export class MaxKey {
+export class MaxKey extends BSONValue {
```

Comment:
does this work without a constructor/super call?

Reply:
Yep, constructor is optional, if one is added both JS runtime / TS will require the author to call super first.